notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -194,7 +194,7 @@
         "    \n",
```

Comment:
<p>Remove this:</p><p>In your case, output from batch prediction job is</p><p><strong>{"instance": {"dense_input": [0.02715405449271202, 0.0, 0.027177177369594574, 0.0, 0.0010195195209234953, 0.009660660289227962, 0.1501501500606537, 0.0027548049110919237, 0.036036036908626556, 1.0, 0.03033033013343811, 0.04091591760516167, 0.043618619441986084]}, "prediction": [0.522156954]}</strong></p><p> </p><p>output of batch prediction job is&nbsp;<strong>"prediction": [0.522156954]</strong>, -&gt;</p><p>output of batch prediction job is&nbsp;<strong>"prediction": [value]</strong>,</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>DONE</p>